docs/api/service-index.md

```diff
@@ -65,6 +65,10 @@ There is no requirement that each resource has a unique `@id` or `@type`. It is
 determine which resource to prefer over another. One possible implementation is that resources of the same or
 compatible `@type` can be used in a round-robin fashion in case of connection failure or server error.
 
+A resource can use a different host or domain than the service index, but this may cause issues in environments with strict network rules.
+If you delegate resources to api.nuget.org, your feed will not work where access to nuget.org is blocked.
+We recommend configuration to enable or disable features that use api.nuget.org.
```

Comment:
I am not sure what this means. I think I can guess it, but maybe it should be expanded a bit with an example. It seems to me like the guidance allows proxying/caching nuget.org (since this would avoid network access controls) which could still be considered "using" api.nuget.org.

I might be missing the context here and that's why I am confused.

Reply:
I mean putting api.nuget.org URLs in the service index, which causes clients to try to access api.nuget.org directly. proxying/caching is great, and what feeds should do, to avoid situations where the server is installed on-prem, and the company firewall blocks nuget.org.

I've updated the PR, let me know what you think now.